src/views/design/ScriptDetail/EditAction/index.tsx

```diff
@@ -226,26 +240,44 @@ function EditAction({ onClose, action, onEdit, state }: IPublicProps & IObserveP
                             onChange={() => {
                                 setErrorExpectedChecked(!errorExpectedChecked);
                             }}
+                            disabled={!checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC')}
```

Comment:
Can we already use the security group name  of the script in question instead of the fixed value 'PUBLIC'